content/en/docs/concepts/architecture/nodes.md

```diff
@@ -274,10 +274,10 @@ availability of each node, and to take action when failures are detected.
 For nodes there are two forms of heartbeats:
 
 * updates to the `.status` of a Node
-* [Lease](/docs/reference/kubernetes-api/cluster-resources/lease-v1/) objects
+* [Lease](/docs/concepts/architecture/leases/) objects
   within the `kube-node-lease`
   {{< glossary_tooltip term_id="namespace" text="namespace">}}.
-  Each Node has an associated Lease object.
+  Each Node has an associated Lease.
```

Comment:
I just wanted to clarify that we intentionally intend to delete the object here because, in the doc below, we have mentioned that a lease object is associated with each node in the `kube-node-lease` namespace.